lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskCommentServiceImpl.java

```diff
@@ -22,19 +22,14 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TaskCommentServiceImpl.class);
 
-  private static final String NOT_AUTHORIZED =
```

Comment:
Why did you delete these? Sonarcloud was not happy without those

Reply:
"We never want String constants for Exception messages or logging statements. We prefer to duplicate those messages here because of code readability" from our guidelines